### PR TITLE
Created a palindrome number function. Needs to get bug fixed.

### PR DESCRIPTION
**Pull request by ystartc**

### CODE REVIEW

File: functions/palindrome_number.py [hunk 1]
```diff
@@ -1,2 +1,9 @@
...
-    pass
+def palindrome_number(number):
+    str_num = str(number)
+    end_index = len(str_num) - 1
+    for i in range(len(str_num)//2):
+        str_num[i] == str_num[end_index]
+        end_index -= 1
+        
+assert palindrome_number(123321) == True
+assert palindrome_number(123322) == False
```
Comment (JenniferWTam) [inline functions/palindrome_number.py:9]: You can add these to the test file, tests for edge cases?

File: functions/palindrome_number.py [hunk 2]
```diff
@@ -1,2 +1,9 @@
-def palindrome_number(x):
-    pass
+def palindrome_number(number):
+    str_num = str(number)
+    end_index = len(str_num) - 1
+    for i in range(len(str_num)//2):
```
Comment (JenniferWTam) [inline functions/palindrome_number.py:4]: How could you do this recursively

**Review by JenniferWTam — CHANGES_REQUESTED** [review]
